Avatar Component/Avatar-documentation.css

```diff
@@ -0,0 +1,22 @@
+iframe{
+    min-width: 60rem;
+    min-height: 39rem;
+}
+
+article{
+    flex-grow: 1;
+    font-size: 1.5rem;
+}
+
+.avatar-heading{
+    font-size: 2rem;
+}
```

Comment:
These styles are getting repeated so consider moving them to global CSS.

Reply:
This style is specific to the documentation site thats why I have not added it in the all component css, hope that is fine.